src/main/scala/com/wavesplatform/transaction/smart/script/v1/ScriptV1.scala

```diff
@@ -50,4 +70,5 @@ case class ScriptV2(version: Version, expr: Contract) extends Script {
       val s = Array(version.toByte) ++ ContractSerDe.serialize(expr)
       ByteStr(s ++ crypto.secureHash(s).take(checksumLength))
     }
+  override val maxBlockVersion: Coeval[Int] = Coeval.evalOnce(2)
```

Comment:
still, 1 and 2 are magic numbers, maybe we don't need the actual version but rather
```
override val containsBlockV2: Coeval[Boolean]
```
?